Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveHeightValueConverter.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    internal class AdaptiveHeightValueConverter : IValueConverter
```

Comment:
If someone wants to adapt the adaptive grid view, won't they need access to this?

Should we instead make it public but in a sub-namespace like `Microsoft.Toolkit.Uwp.UI.Controls.AdaptiveGridView`?

Reply:
I don't see any reason it needs to be public. It's used within a private method. Can always open it at a later date